packages/code-du-travail-frontend/src/contributions/Contribution.js

```diff
@@ -57,51 +57,103 @@ const components = {
 const getConventionSlug = ({ num, title }) =>
   slugify(`${num}-${title}`.substring(0, 80));
 
-const LinkConvention = ({ num, title }) => {
+const LinkConvention = ({ num, title, style }) => {
   const slugConvention = getConventionSlug({ num, title });
   return (
     <Link
       href="/convention-collective/[slug]"
       as={`/convention-collective/${slugConvention}`}
     >
-      <Button variant="secondary">
-        Consulter la convention collective complète
+      <Button style={style} variant="secondary">
+        Consulter la convention collective
       </Button>
     </Link>
   );
 };
 
-const AnswerConvention = ({ markdown }) => (
-  <div
-    style={{
-      backgroundColor: "white",
-      padding: 10,
-      marginBottom: 20
-    }}
-  >
-    <Mdx markdown={makeArticlesLinks(markdown)} components={components} />
-  </div>
+const RefLink = ({ value, url }) => (
+  <LineRef>
+    <a href={url} target="_blank" rel="noopener noreferrer">
+      {value}
+    </a>
+  </LineRef>
 );
 
+const References = ({ references }) => {
+  const agreementRefs =
+    (references && references.filter(ref => !!ref.agreement)) || [];
+  const othersRefs =
+    (references && references.filter(ref => !ref.agreement)) || [];
+
+  return (
+    <React.Fragment>
```

Comment:
Is there an advantage to use React.Fragment if you don't have attribute? 
It's more clear to use <></>

Reply:
i dont think so, looks more cryptic to use `<></>`  imho but i dont mind